messenger.rst

```diff
@@ -653,6 +653,20 @@ config and start your workers:
 
 See the `Supervisor docs`_ for more details.
 
+Graceful Shutdown
+~~~~~~~~~~~~~~~~~
+
+Workers can handle the ``SIGTERM`` POSIX signal to finish handling their
+current message before exiting, as long as the `PCNTL`_ extension is installed.
+
+.. tip::
+
+    In some cases the ``SIGTERM`` signal is sent by Supervisor itself (e.g.
```

Comment:
Is it worth adding an example code ?

Reply:
For the `stopwaitsecs` ? I don't think so, it is only a line like `stopwaitsecs=20` in a `[program:x]` section.